content/en/docs/zero-code/obi/setup/standalone.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Run OBI as a standalone process
+linkTitle: Standalone
+description: Learn how to setup and run OBI as a standalone Linux process.
+weight: 4
+---
+
+OBI can run as a standalone Linux OS process with elevated privileges that can
+inspect other running processes.
+
+## Download and install
+
+You can download the OBI executable from the
+[OBI releases page](https://github.com/grafana/beyla/releases).
```

Comment:
```suggestion
[OBI releases page](https://github.com/open-telemetry/opentelemetry-ebpf-instrumentation/releases).
```

No releases so far, yet I suggest we replace this URL already?

Reply:
That's Ok! At some point it will have releases.